pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authorization/PulsarAuthorizationProvider.java

```diff
@@ -161,6 +162,9 @@ public CompletableFuture<Boolean> canConsumeAsync(TopicName topicName, String ro
     @Override
     public CompletableFuture<Boolean> canLookupAsync(TopicName topicName, String role,
             AuthenticationDataSource authenticationData) {
+        if (SystemTopicNames.isTransactionCoordinatorAssign(topicName)) {
+            return CompletableFuture.completedFuture(true);
+        }
```

Comment:
I am concerned that this design change could lead to other system topics getting this kind of special treatment. Ultimately, the fact that the transaction coordinator topic is needed for all tenants seems like an anti-pattern in our multi-tenant system. I would prefer to add a new auth action for lookup so that operators can choose to expose this to individual tenants.

Additionally, it seems odd to me that we would need to grant a role permission to lookup a topic without the need to produce or consume from that topic.

Reply:
Don't worry. We check the topic name when creating a topic.

> Additionally, it seems odd to me that we would need to grant a role permission to lookup a topic without the need to produce or consume from that topic.

We don't have the lookup permission name. Right now we check whether a user has the consume of produce permission, if so, we allow the user to continue the next operation.